### PR TITLE
fix(server_status): `lsp.inlay_hint.is_enabled expects table`

### PR DESCRIPTION
**Pull request by blob42**
Signed-off-by: blob42 <contact@blob42.xyz>

### CODE REVIEW

File: lua/rustaceanvim/server_status.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ function M.handler(_, result, ctx, _)
   -- as soon as rust-analyzer has fully initialized.
   if type(vim.lsp.inlay_hint) == 'table' then
     for _, bufnr in ipairs(vim.lsp.get_buffers_by_client_id(ctx.client_id)) do
-      if vim.lsp.inlay_hint.is_enabled(bufnr) then
+      if vim.lsp.inlay_hint.is_enabled({bufnr}) then
```
Comment (mrcjkb) [inline lua/rustaceanvim/server_status.lua:21]:
```suggestion
      if vim.lsp.inlay_hint.is_enabled { bufnr = bufnr } then
```

**Review by mrcjkb — COMMENTED** [review]
Thanks 🙏

**Comment by mrcjkb** [comment]
Your fork is out of date. The fix is alre

**Comment by blob42** [comment]
> Your fork is out of date. The fix is already present in the latest version.

Oh sorry I didn't notice, I pushed the PR by reflex :)

**Pull request closed without merging**